src/components/previewers/VideoPlayer.jsx

```
@@ -0,0 +1,180 @@
+                          onClick={() => setSeeMore(!seeMore)}
+                        >
+                          {seeMore ? "See less" : "See more"}
+                        </span>
+                      </>
+                    ) : (
+                      description
+                    )}
+                  </div>
+                </div>
+              </div>
+              <div className="w-full h-[1px] bg-gray-200 my-2"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)